systemtest/src/test/java/io/strimzi/systemtest/operators/user/UserScalabilityIsolatedST.java

```diff
@@ -0,0 +1,148 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.systemtest.operators.user;
+
+import io.strimzi.api.kafka.model.AclOperation;
+import io.strimzi.api.kafka.model.KafkaUser;
+import io.strimzi.api.kafka.model.KafkaUserAuthorizationSimple;
+import io.strimzi.api.kafka.model.KafkaUserAuthorizationSimpleBuilder;
+import io.strimzi.api.kafka.model.KafkaUserBuilder;
+import io.strimzi.api.kafka.model.KafkaUserSpec;
+import io.strimzi.systemtest.AbstractST;
+import io.strimzi.systemtest.annotations.IsolatedTest;
+import io.strimzi.systemtest.annotations.KRaftNotSupported;
+import io.strimzi.systemtest.enums.UserAuthType;
+import io.strimzi.systemtest.storage.TestStorage;
+import io.strimzi.systemtest.templates.crd.KafkaTemplates;
+import io.strimzi.systemtest.templates.crd.KafkaTopicTemplates;
+import io.strimzi.systemtest.templates.crd.KafkaUserTemplates;
+import io.strimzi.systemtest.utils.kafkaUtils.KafkaUserUtils;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.extension.ExtensionContext;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static io.strimzi.systemtest.Constants.SCALABILITY;
+
+@Tag(SCALABILITY)
+public class UserScalabilityIsolatedST extends AbstractST {
+    private static final Logger LOGGER = LogManager.getLogger(UserScalabilityIsolatedST.class);
+    private static String clusterName = "";
```

Comment:
Why do you explicitly init these variables (i.e., cluster and topic name) with empty string? By default, these string vars will have null is there any hidden logic  I am overlooking? :)

Reply:
No big logic behind it, I just didn't want to have it `null`. I will change it 😄  thanks